app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleSearch/SearchPopover.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <div
+    class="fixed flex items-center justify-center w-screen h-screen bg-white/70 top-0 left-0 z-50"
+  >
+    <div
+      v-on-clickaway="onClose"
+      class="flex flex-col px-4 pb-4 rounded-md shadow-md border border-solid border-slate-50 dark:border-slate-800 bg-white dark:bg-slate-900 z-[1000] max-w-[720px] md:w-[20rem] lg:w-[24rem] xl:w-[28rem] 2xl:w-[32rem] h-[calc(100vh-20rem)] max-h-[40rem]"
+    >
+      <search-header
+        :title="$t('HELP_CENTER.ARTICLE_SEARCH.TITLE')"
+        class="w-full sticky top-0 bg-[inherit]"
+        @close="onClose"
+        @search="onSearch"
+      />
+
+      <article-view
+        v-if="activeId"
+        :url="articleViewerUrl"
+        @back="onBack"
+        @insert="onInsert"
+      />
+      <search-results
+        v-else
+        :search-query="searchQuery"
+        :is-loading="isLoading"
+        :portal-slug="selectedPortalSlug"
+        :articles="searchResultsWithUrl"
+        @preview="handlePreview"
+        @insert="onInsert"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import { debounce } from '@chatwoot/utils';
+import { mixin as clickaway } from 'vue-clickaway';
+import {
+  isEscape,
+  isActiveElementTypeable,
+} from 'shared/helpers/KeyboardHelpers';
+
+import SearchHeader from './Header.vue';
+import SearchResults from './SearchResults.vue';
+import ArticleView from './ArticleView.vue';
+import ArticlesAPI from 'dashboard/api/helpCenter/articles';
+import portalMixin from '../../mixins/portalMixin';
+
+export default {
+  name: 'ArticleSearchPopover',
+  components: {
+    SearchHeader,
+    SearchResults,
+    ArticleView,
+  },
+  mixins: [clickaway, portalMixin],
+  props: {
+    selectedPortalSlug: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      searchQuery: '',
+      isLoading: false,
+      searchResults: [],
+      activeId: '',
+      debounceSearch: () => {},
+    };
+  },
+  computed: {
+    articleViewerUrl() {
+      const article = this.activeArticle(this.activeId);
+      if (!article) return undefined;
+      const isDark = document.body.classList.contains('dark');
```

Comment:
Please use the darkMode getter here.

Reply:
We are not using this state in dashboard.